server/src/main/java/org/apache/iotdb/db/metadata/mtree/IMTreeBelowSG.java

```diff
@@ -33,7 +33,7 @@
 import java.util.Map;
 import java.util.Set;
 
-public interface IMTreeBelowSG {
+public interface IMTreeBelowSG<N extends IMNode<N>> {
```

Comment:
Should this class be eliminated? Seems useless.

Reply:
Add todo. Lets eliminate it in another PR.